File: PMUG02_16_2021_Project/Start_Java/app/src/main/java/com/neudesic/pmugfebappstart/viewmodel/PersonViewModel.java
package com.neudesic.pmugfebappstart.viewmodel;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;

import com.neudesic.pmugfebappstart.model.Person;

import java.util.List;

public class PersonViewModel extends AndroidViewModel {
    private final LiveData<List<Person>> personsData = new MutableLiveData<>();

    public PersonViewModel(@NonNull Application application) {
        super(application);

        //instantiate the room database here
    }

    public LiveData<List<Person>> getData() {
        return personsData;
    }

    public void addPerson(String name, int age, String birthday) {
        //add method
    }
}

File: PMUG02_16_2021_Project/Start_Kotlin/app/src/main/java/com/neudesic/pmug02_16_2021_kotlin/viewmodel/PersonViewModel.kt
package com.neudesic.pmug02_16_2021_kotlin.viewmodel

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import com.neudesic.pmug02_16_2021_kotlin.model.Person

class PersonViewModel(app: Application) : AndroidViewModel(app) {
    private val personsData: MutableLiveData<List<Person>> = MutableLiveData()

    val data: LiveData<List<Person>> = personsData

    init {
        //instantiate the room database here
    }

}